### ISSUE TITLE
KCN Incentive

### ISSUE THREAD

**Issue by BitSwift-v2** [sacha-l/kingston-community-network#11]
The KCN will be equipped with its own Community based digital asset that will provide the incentive layer for the network.  The incentive layer will target both businesses partipating and the users using the KCN.  Ideally this would come in the form of a digital token which is accepted across all the businesses who are participating in the KCN.  This same token would be issued to users on a regular and consistent basis as part of the incentive mechanism.  There is lots to think about here.

**Comment by sacha-l** [comment]
Yes. Lots to consider. However, starting out with observing how users interact with the splash page during the "trial phase" is a good place to start. Let's keep this issue open to capture incoming ideas we'd want to implement as the trial phase goes by. 

I'm thinking we should give users 'KCN cash' rewards for shopping local. This cash can be spent at participating KCN businesses. Simple.
